libs/backend/src/ballot_package/ballot_package_io.test.ts

```diff
@@ -25,6 +34,18 @@ import { readBallotPackageFromUsb } from './ballot_package_io';
 import { createMockUsb } from '../mock_usb';
 import { UsbDrive } from '../get_usb_drives';
 
+jest.mock('@votingworks/utils', (): typeof import('@votingworks/utils') => ({
+  ...jest.requireActual('@votingworks/utils'),
+  isFeatureFlagEnabled: jest.fn(),
```

Comment:
you can use the `FeatureFlagMock` object from `features.ts` in `libs/utils` if it makes this any easier

Reply:
Nice! Used it here and in `libs/auth`: https://github.com/votingworks/vxsuite/pull/3502/commits/6bce14c55551d1b68d24647cf2a6d424f80e3619